modules/virt-importing-vm-datavolume.adoc

```diff
@@ -105,8 +111,8 @@ status: {}
 ----
 <1> Specify the name of the virtual machine.
 <2> Specify the name of the data volume.
-<3> Specify `http` for an HTTP or HTTPS endpoint. Specify `registry` for a container disk image imported from a registry.
-<4> Specify the URL or registry endpoint of the virtual machine image you want to import. This example references a virtual machine image at an HTTPS endpoint. An example of a container registry endpoint is `url: "docker://kubevirt/fedora-cloud-container-disk-demo:latest"`.
+<3> The volume and access mode are detected automatically for known storage provisioners. Alternatively, you can specify Block.
```

Comment:
I don't see an access mode in the yaml, nor specified by the `<3>` tag. Is volume and access mode the same thing? I don't think it would be necessary to mention if its not specified in the yaml. I might not be understanding it right though, WDYT?

Reply:
I asked the SME about this, and he said, "So for VMs we have a special `storage` field instead of `pvc` if the user uses that (which the example does) you don't have to provide accessMode or volumeMode and these values will be automatically inferred from the StorageProfile. You can specify them to override the values inferred. So technically you don't need the volumeMode in the example either, but we are talking explicitly about block volumes below, which is why it is specified I guess." So I think the YAML is okay as is.